### ISSUE TITLE
Reduced beta-hexosaminidase (HEX) activity

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7026]
**Preferred term label:**
Reduced beta-hexosaminidase activity

**Synonyms**

**Definition (free text, please give PubMed ID)**
Diminished activity of the enzyme beta-hexosaminidase in the blood circulation.

**Parent term (use hpo.jax.org/app)**
Abnormal enzyme/coenzyme activity HP:0012379

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 268800 
Sandhoff disease
PMID:33407268 5/5

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
